src/app/meetups/meetups.component.ts

```diff
@@ -3,6 +3,7 @@ import { CouchService } from '../shared/couchdb.service';
 
 @Component({
   templateUrl: './meetups.component.html',
+  providers:[CouchService]

```

Comment:
This `providers` line should go into the `configureTestingModule` in the `meetups.component.spec.ts` file.

I had to read up to double check, but this actually instantiates a new `CouchService` class.  While at this point it wouldn't break the app (the class isn't holding a shared state across components), for now we should stick with having a singleton service to connect to the DB.